### ISSUE TITLE
Result list import/export

### ISSUE THREAD

**Issue by aminya** [krausest/js-framework-benchmark#956]
I always want to compare a set of frameworks to each other and ignore everything else. But this is currently time-consuming, as I should click the frameworks one by one each time.

I have written a simple script that automates my work, but I think there should be an import/export configuration that allows saving the state of the benchmark list.

```ts
const frameworks = [
    "angular-v12.0.1",
    "elm-v0.19.1-3",
    "inferno-v7.4.8",
    "lit-html-v1.1.0",
    "preact-v10.5.14",
    "react-v17.0.1",
    "react-hooks-v17.0.1",
    "solid-v1.1.0",
    "stencil-v2.6.0",
    "svelte-v3.42.1",
    "vanillajs",
    "vue-v3.2.1",
];

document.querySelector("#root > div > div.selectBar > div > div:nth-child(1) > button").click();

document.querySelector("#root > div > div.selectBar > div > div.open.dropdown-container > div.show.dropdown-menu > div:nth-child(1) > div.float-rt > button:nth-child(1)"
).click();

[
    ...document.querySelectorAll(
        "#root > div > div.selectBar > div > div.open.dropdown-container > div.show.dropdown-menu > div:nth-child(1) > div.grid > div"
    ),
]
    .filter((elm) => frameworks.includes(elm.children[1].textContent))
    .forEach((elm) => elm.firstElementChild.click());

```

**Comment by krausest** [comment]
Would storing the previous frameworks and benchmarks selection in the browser's localstorage help?

**Comment by aminya** [comment]
I think the state should be kept across different sessions.

**Comment by krausest** [comment]
Keeping the state in localstorage would do exactly that, or do I get you wrong?

**Comment by aminya** [comment]
Yes, as long as it is later restored from the localstorage.

**Comment by krausest** [comment]
Fixed with #1048
